src/Microsoft.AspNetCore.BuildTools.ApiCheck/build/Common.props

```diff
@@ -0,0 +1,15 @@
+﻿<!--
+  WARNING: These targets are intended for building Microsoft's ASP.NET Core repos and are not intended
+  for use outside of Microsoft.
+-->
+<Project>
+  <!-- Set nothing if handled in project file or already done in outer build. -->
+  <PropertyGroup>
+    <EnableApiCheck Condition=" '$(EnableApiCheck)' == '' AND '$(MSBuildRuntimeType)' == 'Core' ">true</EnableApiCheck>
```

Comment:
It was discussed w/ the engineering team that we don't want this to run when building from VS.